stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3821 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        # We need a column for each 3-tuple of team, map, and win/loss, counting how many
</pre><p>The creation of all the columns and what they mean is tricky to get your head around (here and to a lesser extent the equivalent functions for the next two models as well). I don't suggest changing the code, but maybe it's worth acknowledging that it is tricky and maybe pointing people to look at the column in the resulting data frame underneath to help figure out what's going on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>